lighthouse-core/test/audits/metrics-test.js

```diff
@@ -11,6 +11,9 @@ const TTIComputed = require('../../computed/metrics/interactive.js');
 const pwaTrace = require('../fixtures/traces/progressive-app-m60.json');
 const pwaDevtoolsLog = require('../fixtures/traces/progressive-app-m60.devtools.log.json');
 
+const lcpTrace = require('../fixtures/traces/lcp-m79.json');
```

Comment:
oh! i almost forgot, you should be able to run these guys through the trace and devtoolslog minifier scripts to clean up them and make the diff not so 30k lines scary :) that's what I did for traces in #9727

Reply:
expected that the observed values would change a bit?

![image](https://user-images.githubusercontent.com/4071474/65926372-dee46900-e3a9-11e9-9657-4f13f71f45e2.png)

Reply:
speed index and trace end yes :) 

the minor change to speed index is because of screenshot throttling in the minify script and trace end is because the real trace end was a non-essential event so it's not a big deal 👍